src/Cassandra/Exceptions/NoHostAvailableException.cs

```diff
@@ -18,6 +18,7 @@
 using System.Collections.Generic;
 using System.Net;
 using System.Runtime.Serialization;
+using System.Text;
 
 namespace Cassandra
```

Comment:
Wdyt about change the namespace to Cassandra.Exceptions (to match the folder structure)? I don't have a strong options about it, but I think it could be clearer to user when using code completion

Reply:
It would be nice to have the exceptions on the correct namespace, unfortunately the exceptions were originally exposed on this namespace and changing it would be a breaking change...

Reply:
Agree, it would have to be a long term refactoring (create inherited classes on "right namespace" and mark the current ones as deprecated, etc...)
👍 on keeping the current namespace.